backend/internal/model/handler/http/v1/model.go

```diff
@@ -199,7 +201,7 @@ func (h *ModelHandler) GetTokenUsage(c *web.Context, req domain.GetTokenUsageReq
 //	@Success		200		{object}	web.Resp{data=domain.GetProviderModelListResp}
 //	@Router			/api/v1/model/provider/supported [get]
```

Comment:
GetProviderModelList 方法同样需要改为使用实例方法调用，确保 ModelList 方法签名一致。

```suggestion
建议添加注释说明方法调用的变更：

```go
// 从 modelkit.ModelList 改为 h.modelkit.ModelList
modelkitRes, err := h.modelkit.ModelList(c.Request().Context(), &modelkitDomain.ModelListReq{
    // ... 参数
})
```